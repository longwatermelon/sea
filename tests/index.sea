fn main() -> int {
    let arr: int* = stalloc(2,int);
    arr[0]=1;
    arr[1]=101;
    return arr[0]+arr[1];
}
