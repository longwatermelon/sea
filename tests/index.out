RETURN_CODE:0
STDOUT:102
